source/operations/data-recovery/recover-after-site-failure.rst

```diff
@@ -1,13 +1,76 @@
 .. _minio-restore-hardware-failure-site:
 
-==========================
-Recover after Site Failure
-==========================
+=====================
+Site Failure Recovery
+=====================
 
 .. default-domain:: minio
 
 .. contents:: Table of Contents
    :local:
    :depth: 1
 
-ToDo- site replication recovery procedure
+MinIO can make the loss of an entire site, while significant, a relatively minor incident.
+Site recovery depends on the backup option you use for the site.
```

Comment:
I understand the intent here, but the second sentence loses me since we're talking about 'backup options' - which many users will assume is traditional backup, and not replication setup.

Since we will have around 3 distinct types of replication a user could configure, we should focus on that and briefly summarize (maybe as a table) what kinds of site failures each replication type can work with.

For example:

- Site Replication : total restoration from healthy peer site
- Bucket Replication : data restoration (objects only) per configured bucket from healthy remote site
- Other REplication; Whatever limitations it might have

Reply:
That's a really good call out. Fixed.